pkg/virt-launcher/virtwrap/BUILD.bazel

```diff
@@ -71,7 +71,9 @@ go_test(
         "nichotplug_test.go",
         "virtwrap_suite_test.go",
     ],
+    data = glob(["testdata/**"]),
     embed = [":go_default_library"],
+    embedsrcs = ["testdata/migration_domain.xml"],
```

Comment:
I don't think you need to have both ` data = glob(["testdata/**"])` and `embedsrcs = ["testdata/migration_domain.xml"]`. AFAIK, `data = ...` is needed only if you want to reference the file in runtime:

https://github.com/kubevirt/kubevirt/blob/d5485fd59023c8bff60b1c7f9cfb55a1c061906c/pkg/virt-handler/node-labeller/capabilities_test.go#L16

IMHO, `embedsrc` should be enough.

Reply:
I did not edit those myself, they were generated by `make bazel-generate` on each per commit.
Testing here, removing it by hand and running `bazel-generate` reintroduces it again.

It is also the same in the other places that uses `testdata`, see:

```bash
toso@tapioca ~/s/k/kubevirt (use-embed)> grep -rniI "glob.*testdata" *
pkg/monitoring/domainstats/downwardmetrics/BUILD.bazel:33:    data = glob(["testdata/**"]),
pkg/virt-handler/isolation/BUILD.bazel:41:    data = glob(["testdata/**"]),
pkg/virt-handler/heartbeat/BUILD.bazel:27:    data = glob(["testdata/**"]),
pkg/virt-handler/node-labeller/BUILD.bazel:42:    data = glob(["testdata/**"]),
pkg/virt-launcher/virtwrap/api/BUILD.bazel:34:    data = glob(["testdata/**"]),
pkg/virt-launcher/virtwrap/converter/BUILD.bazel:49:    data = glob(["testdata/**"]),
pkg/virt-launcher/virtwrap/launchsecurity/BUILD.bazel:17:    data = glob(["testdata/**"]),
pkg/virt-launcher/virtwrap/BUILD.bazel:74:    data = glob(["testdata/**"]),
pkg/downwardmetrics/vhostmd/BUILD.bazel:23:    data = glob(["testdata/**"]),
```

Reply:
So, if `data = ...` is not needed, we should tweak bazel generation and likely apply everywhere. I"d vote for keeping it now for consistency.

Reply:
Okay, I see. I was not aware that it was generated by bazel. Let’s keep it then. Thanks.